src/test/java/org/openstreetmap/atlas/checks/validation/tag/ConflictingCarAccessibilityCheckTestRule.java

```diff
@@ -0,0 +1,218 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+
+/**
+ * {@link ConflictingCarAccessibilityCheck} test data.
+ *
+ * @author sayas01
+ */
+public class ConflictingCarAccessibilityCheckTestRule extends CoreTestRule
+{
+    private static final String TEST_1 = "47.2136626201459,-122.443275382856";
+    private static final String TEST_10 = "47.2136413561665,-122.436028431137";
+    private static final String TEST_12 = "47.2132054427106,-122.44382320858";
+    private static final String TEST_2 = "47.2138327316739,-122.44258668766";
+    private static final String TEST_3 = "47.2136626201459,-122.441897992465";
+    private static final String TEST_4 = "47.2138114677627,-122.440990166979";
+    private static final String TEST_5 = "47.2136200921786,-122.44001973284";
+    private static final String TEST_6 = "47.2135137721113,-122.439127559518";
+    private static final String TEST_7 = "47.2136200921786,-122.438157125378";
+    private static final String TEST_8 = "47.2136413561665,-122.437468430183";
+    private static final String TEST_9 = "47.2137689399148,-122.436717126333";
+    // Atlas to test car navigable edges with car access
+    @TestAtlas(
+            // nodes
+            nodes = { @TestAtlas.Node(coordinates = @TestAtlas.Loc(value = TEST_1)),
```

Comment:
It is a bit to refactor, but this file would look a lot neater if all the `TestAtlas` sub interfaces (ex. @TestAtlas.**Node**) were imported directly (ex. import org.openstreetmap.atlas.utilities.testing.TestAtlas.**Node**;).

Reply:
Right! I have refactored the code to reflect this. Please take a look.